README.rst

```diff
@@ -51,22 +51,58 @@ This library has the following exports:
   language from its dropdown.
 * supportedLanguages: An array of objects representing available languages.  See example below for object shape.
 
+Customization via MFE config API
```

Comment:
I think this description should mention all the possible ways to modify the configuration, I mean, should include the JavaScript File https://github.com/openedx/frontend-platform/blob/master/docs/decisions/0007-javascript-file-configuration.rst